CodeEdit/NavigatorSidebar/SourceControlNavigator/SourceControlSearchToolbar.swift

```diff
@@ -0,0 +1,88 @@
+//
+//  SourceControlSearchToolbar.swift
+//  CodeEdit
+//
+//  Created by Nanashi Li on 2022/05/05.
+//
+
+import SwiftUI
+import CodeEditUI
+
+struct SourceControlSearchToolbar: View {
+
+    @Environment(\.colorScheme)
+    var colorScheme
+
+    @FocusState
+    private var isFocused: Bool
+
+    @Environment(\.controlActiveState)
+    private var controlActive
+
+    @State
+    private var text: String = ""
+
+    @ViewBuilder
+    public func selectionBackground(
+        _ isFocused: Bool = false
+    ) -> some View {
+        if self.controlActive != .inactive {
+            if isFocused {
+                if colorScheme == .light {
+                    Color.white
+                } else {
+                    Color(hex: 0x1e1e1e)
+                }
+            } else {
+                if colorScheme == .light {
+                    Color.black.opacity(0.06)
+                } else {
+                    EffectView(.mediumLight).opacity(0.37)
+                }
+            }
+        } else {
+            if colorScheme == .light {
+                Color.clear
+            } else {
+                EffectView(.mediumLight).opacity(0.14)
```

Comment:
I got a warning `'mediumLight' was deprecated in macOS 10.14: Use a semantic material instead.  To force the appearance of a view hierarchy, set the `appearance` property to an appropriate NSAppearance value.`